airgun/views/computeresource.py

```diff
@@ -199,34 +218,207 @@ class compute_profiles(SatTab):
             }
         )
 
+    class images(SatTab, SearchableViewMixin):
```

Comment:
same here. Could you try to use `Table` instead?

Reply:
This is SatTab and not SatTable

Reply:
oops